src/components/DelegationDashboard/Chat/Messages.tsx

```
@@ -0,0 +1,164 @@
+			button.classList.toggle('opacity-60', !newMessage || loading);
+		}
+	}, [newMessage, loading]);
+
+	return (
+		<div className='flex h-[calc(100%-300px)] w-full flex-col'>
+			<Card
+				className='w-full rounded-none border-t-0'
+				bodyStyle={{ display: 'flex', alignItems: 'center', gap: '0.5rem', width: '100%' }}
+				size='small'
+			>
+				{recipientAddress && recipientAddress?.startsWith('0x') ? (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)